app/views/settings/preferences/show.html.haml

```diff
@@ -44,6 +47,3 @@
 
   .fields-group
     = f.input :setting_auto_play_gif, as: :boolean, wrapper: :with_label
-
-  .actions
-    = f.button :button, t('generic.save_changes'), type: :submit
```

Comment:
I think that it would be better to leave this. I think that it is better to have it both above and below.